css-view-transitions-2/Overview.bs

```diff
@@ -1237,25 +1314,118 @@ When [[css-view-transitions-1#setup-transition-pseudo-elements-algorithm|setting
+
+		1. [=Adjust the nested group transform=] to |transform|, given |groupContainerElement|'s [=transform from snapshot containing block=], |containerRect|'s width |containerRect|s height, and |groupContainerElement|'s [=new box properties=].</div>
+
+## Capturing layered CSS properties ## {#layered-capture-algorithms}
+
+### Compute the layered-capture style ### {#layered-capture-compute-style}
 
-		1. [=Multiply=] |transform| by the inverse matrix of |groupContainerElement|'s [=transform from snapshot containing block=].
+<div algorithm="compute layered capture style">
+To compute the <dfn>layered capture style</dfn> of an {{Element}} |element|:
+
+	1. Let |propertiesToCapture| be a new [=/list=] corresponding to the [=layered capture properties=],
+	1. Issue: Specify the behavior of 'overflow' and containment. See <a href="https://github.com/w3c/csswg-drafts/issues/11079">issue 11079</a>.
+	1. Let |styles| be a « ».
+	1. For each |property| in |propertiesToCapture|, [=list/append=] the [=string/concatenate|concatentation=] of « |property|, `":"`, |element|'s [=computed value=] of |property|, `";"` » to |styles|.
+	1. Return the [=string/concatenate|concatentation=] of |styles|.
+</div>
+
+<div algorithm="compute layered capture geometry">
+To compute the <dfn>layered capture geometry</dfn> of an {{Element}} |element|, return a new [=layered box properties=],
+	whose [=layered box properties/box sizing=] is |element|'s [=computed value|computed=] 'box-sizing',
+	[=layered box properties/padding box=] is |element|'s [=/padding box=],
+	and whose [=layered box properties/content box=] is |element|'s [=/content box=].
+</div>
+
+### Capture the old layered properties ### {#capture-new-layered-props-algorithm}
+<div algorithm="additional old capture steps (layered)">
+When [[css-view-transitions-1#capture-new-state-algorithm|capturing the old state for an element]], perform the following steps given a [=captured element=] |capturedElement| and an [=element=] |element|:
+
+	1. Set |capturedElement|'s [=old layered-capture style=] to |element|'s [=layered capture style=].
+	1. Set |capturedElement|'s [=old box properties=] to |element|'s [=layered capture geometry=].
+</div>
+
+### Adjustment to image capture size ### {#capture-image-size-algorithm}
+<div algorithm="adjust image capture size (layered)">
+When [=capturing the image=] given |element|:
+	1. Let |geometry| be |element|'s [=layered capture geometry=].
+	1. If |geometry| is not null and |geometry|'s [=layered box properties/box sizing=] is ''box-sizing/content-box'',
+		then the [=natural dimensions=] of the image will be based on |geometry|'s [=layered box properties/content box=]'s size rather than on the [=border box=].
+</div>
+
+### Render the snapshot with layered capture ### {#layered-capture-rendering}
+
+When capturing an element into a snapshot, only the [=element contents=] are painted, without the element's effects and box decorations.
```

Comment:
Would love to get @vmpstr's input on this phrasing too.

Taking inspiration from the wording in L1: https://drafts.csswg.org/css-view-transitions-1/#capture-the-image-algorithm:~:text=containing%20block.-,Otherwise,-%3A.
- We want to say what is painted, which is covered here.
- We want to say what the image size is internally in the UA which I think would be the union of ink overflow rectangles of all the descendants + size of replaced content..?
- And we want to define the web observable size of the image which is covered above as content box.